upb/table_internal.h

```diff
@@ -263,6 +263,38 @@ bool upb_strtable_resize(upb_strtable *t, size_t size_lg2, upb_arena *a);
 
 /* Iterators ******************************************************************/
 
+/* New-style iterators.  Much simpler, iterator state is held in size_t.
+ *
+ *   intptr_t iter = UPB_INTTABLE_BEGIN;
```

Comment:
comment says state is a `size_t` but code uses `intptr_t`.

Also the "new" and "much simpler" don't really add much to the comment (and will become out of date as soon as these have settled for a while).

Reply:
Done.